src/components/VmDetails/cards/UtilizationCard/style.css

```diff
@@ -16,6 +16,8 @@
 
   display: flex;
   flex-direction: column;
+
+  justify-content: space-around;
```

Comment:
I like `justify-content: center;` a bit better then `space-around` so that the card body content maintain their spacing when the cards get taller as the NICs or Disks get taller.

![vm portal - google chrome_010](https://user-images.githubusercontent.com/3985964/49900348-51d91f00-fe2c-11e8-952f-bf2221c56c99.png)

![vm portal - google chrome_009](https://user-images.githubusercontent.com/3985964/49900353-556ca600-fe2c-11e8-9d11-7bc4fb9bc5cd.png)

Reply:
With `justify-content: center` there is a lot of free space around chart, for me it looks to very good.

Reply:
@lwrigh , @gregsheremeta - The charts on the utilization and how they change position, or not, when the Nics/Disks need to grow is an open question.  3 simple ways:
  1. Leave the charts at the top of the card and let the white space at the bottom take up the extra space (like in Laura's mockup in comment https://github.com/oVirt/ovirt-web-ui/pull/890#issuecomment-447017958)
  2. Have the card contents stretch and realign the elements to space out (as in justify-content: space-around)
  3. Have the card contents stretch but keep the elements relative position.  White space at top between the card title and content, and the white space below the contents grows equally.  (as in justify-content: center)

And last thought, the "No Data Available" already grows like #3 so it would need to be tweaked to accomplish #1.

Reply:
@sjd78 so we went with option 1 right?

Reply:
> @sjd78 so we went with option 1 right?

Correct.